content/pl/build-release-run.md

```diff
@@ -0,0 +1,18 @@
+## V. Buduj, wypuszczaj, uruchamiaj
+### Precyzyjnie oddziel etap budowania od uruchamiania
+
+[Codebase](./codebase) jest przetwarzany w (nie developerskie) wdrożenie w trzech etapach
+
+* Podczas *etapu budowania* kod z repozytorium jest konwertowany do wykonywalnej paczki tzw. *buildu*. Używając wersji kodu zdefiniowanej przez commit w procesie deploymentu, w etapie budowania aplikacji pobiera się, dołącza do projektu [zależności](./dependencies) i kompiluje niezbędne zasoby.
+* W *etapie wypuszczania* aplikacji brany jest build stworzony w poprzednim etapie i konfigurowany na bazie [ustawień](./config) obecnego wdrożenia.  Stworzony w ten sposób *release* zawiera zbudowane źródło kodu, jego konfigurację i jest gotowe do uruchomienia w wybranym środowisku.
+* *Etap uruchamiania* (znany również jako "runtime") odpala aplikację w środowisku wykonawczym przez uruchomienie zestawu [procesów](./processes) opisanych w przygotowanym releasie.
+
+![Kod staje się buildem, jeśli zostanie połączony z konfiguracją by stworzyć release](/images/release.png)
+
+**Aplikacja dwunastu aspektów ścieśle rozgranicza etapy budowy, wydania i uruchomienia**  Nie jest więc możliwe zrobić zmian kodu podczas działania aplikacji, ponieważ nie można przenieść ich z powrotem do etapu budowy.
+
+Narzędzia do obsługi wdrożeń zazwyczaj oferują moduły do zarządzania releasami, w tym możliwość do powrotu do poprzedniej wersji (roll back). Np. narzędzie [Capistrano](https://github.com/capistrano/capistrano/wiki) przechowuje releasy w podkatalogu `releases`, gdzie obecna wersja releasu jest symlinkowana do tej z katalogu. Komenda `rollback` pozwala na szybką zmianę  wypuszczonej aplikacji na jedną z poprzednich.
+
+Każdy release powinien zawsze posiadać unikalny identyfikator, jak np. timestamp wypuszczenia aplikacji (taki jak `2011-04-06-20:32:17`) lub inkrementowany numer (np. `v100`). Do rejestru wypuszczonych wersji aplikacji można jedynie dodawać jej nowe wersje, a jego zawartość nie może być zmieniana w żaden inny sposób. Jedynym sposobem na zmianę wypuszczonej aplikacji wypuszczenie nowej.
```

Comment:
1. W tym rozdziale chyba możesz używać `wydanie` zamiast `release`. 
2. Timestamp w tym kontekście można tłumaczyć jako `datę wypuszczenia...` (i znowu to nieszczęsne "wypuszczenie")
3. `Jedynym sposobem na zmianę wypuszczonej aplikacji wypuszczenie nowej.` -> masło maślane, Kali pić, Kali spać itd. ;)